### PR TITLE
 #127 Provide instructions for verifying release

### PR DESCRIPTION
**Pull request by mikewalch**
* Adding instructions for running integrations
* Will add more instructions for running example applications

### CODE REVIEW

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: Replace with less wordy, and more personal: "You can test a release by:"

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: No need for both a bulleted list here, and then the sections below. It is redundant, and wordy. This bulleted list can go away completely.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: The user can see that. There's no reason for this sentence.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: This sentence isn't necessary.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: Too wordy. Better to use a system property in the below profile, and have the user set it on the command line.
This can become:
```
1. Set the version number: $ export RCVERSION=1.2.0
2. Set the staging repository URL: $ export STAGINGREPO=https://...orgapachefluo-1023/
```

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: This is far too wordy and explanatory. Just put the command to create a temporary settings.xml file, which can be specified on the maven command line later:
```
3. Create a settings.xml file: $ cat <<EOF >/tmp/fluo-its-settings.xml
   ...
EOF
```

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: No need for sentence. Just put the command.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: Shorten to: `Build with: $ mvn -s /tmp/fluo-its-settings.xml clean verify -Dfluo.version=$RCVERSION`
Get rid of preceding paragraph which is redundant with the command.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: No need for this if using a separate settings.xml file.

**Review by ctubbsii — COMMENTED** [review]
I like the idea, but I dislike how wordy it is. I think it should be far less descriptive, and much more "copy/paste". I've provided some suggested changes, but am flexible. I really do think it should be bare minimum, though.

**Comment by mikewalch** [comment]
@ctubbsii, I made a bunch of updates. Let me know what you think.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: Since these instructions show using `/tmp`, there's no reason to suspect that the previous file will still exist against the new RC. It'd probably be better to just have them start over. That could simplify some of this, because there'd be no reason to put the filename in a variable, or to use `\${env.RC_STAGING}` (could use `$RC_STAGING`) instead... or better yet... put most of the URL in the file, and just the number in the variable.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: A sed command would be better here than vim instructions.

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md:227]: Is there more to test here, or just testing that uno can setup fluo?

File: pages/release-process.md
Comment (ctubbsii) [inline pages/release-process.md]: I think this section should go after the phrasecount example with uno. It's really a separate test case, and it is breaking up the uno test case with phrasecount.

**Review by ctubbsii — APPROVED** [review]
I like this version better. I have additional ideas, but I think it might be easier if I submitted my own PR to make the changes in my head, rather than hold up your changes with my feedback.

File: pages/release-process.md
Comment (mikewalch) [reply pages/release-process.md:227]: This is set up instructions for uno which is need to run example applications. I pushed a commit to make this more clear

File: pages/release-process.md
Comment (mikewalch) [reply pages/release-process.md]: I reorganized it a little better.  I move this before the the uno / phrasecount examples.

File: pages/release-process.md
Comment (keith-turner) [inline pages/release-process.md:221]: this is nice

File: pages/release-process.md
Comment (mikewalch) [reply pages/release-process.md]: I can do that `RC_STAGING` will just be the number (i.e `1023`)

Comment (mikewalch) [reply pages/release-process.md]: I like the idea of using the staging repo number. I pushed a commit that does this.

**Comment by mikewalch** [comment]
I pushed some more commits after this was approved.  Is it OK still I merge it?

**Pull request merged**
